packages/@aws-cdk/aws-custom-resource-sdk-adapter/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@aws-cdk/sdk-v2-to-v3-adapter",
-  "description": "Adapter to convert AWS SDK v2 to AWS CDK v3 calls",
+  "name": "@aws-cdk/aws-custom-resource-sdk-adapter",
+  "description": "Adapter to convert AWS SDK v2 to AWS CDK v3 calls for AwsCustomResource",
```

Comment:
nit: can we update this description to no longer refer to this as converting v2 to v3 calls, but maybe something like Adapter to convert AWS CDK to AWS SDK compatible calls (or something more generic)?